backend/github.js

```diff
@@ -53,3 +53,26 @@ exports.retrieveOrganizations = function (accessToken, callback) {
     callback(null, result);
   });
 };
+
+/**
+ * Delete a Repository WebHook
+ * @param name: name of the Repository
+ * @param hook: Hook Id
+ * @param accessToken: Access Token of the user
+ * @param callback
+ */
+exports.deleteHook = function (name, hook, accessToken, callback) {
```

Comment:
can we shift to Promises from callback? consistency will be followed everywhere. we can't avoid the use of Promises but then we can avoid the use of the callback.